### PR TITLE
Fixes #24045: Add initial release notes for 8.1

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/24045

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by VinceMacBuche** [comment]
Select screen in technique editor can be updated !!

**Comment by amousset** [comment]
All screenshots will need to be updated, some of them were from the "early bootstrap upgrade" era.

**Review by VinceMacBuche — APPROVED** [review]

File: release-data/changelogs/8.1/index.adoc [hunk 1]
```diff
@@ -1,6 +1,193 @@
...
+=== Multi-tenant server
+
+We are introducing a major new concept in Rudder ACLs, representing
+different teams working on the same Rudder server but
+operating different sets of nodes, called tenants.
+
+A Rudder server can be used by different tenants, which are
+defined as a set of users and a set of node groups.
+These users will only be able to read information about the nodes part of their tenant.
+A dedicated interface is also part of the feature, allowinf to manage the tenants of the server.
```
Comment (fanf) [inline release-data/changelogs/8.1/index.adoc:114]: Allowing

File: release-data/changelogs/8.1/index.adoc [hunk 2]
```diff
@@ -1,6 +1,193 @@
...
+== Identity and Access Management
+
+=== Multi-tenant server
+
+We are introducing a major new concept in Rudder ACLs, representing
+different teams working on the same Rudder server but
+operating different sets of nodes, called tenants.
+
+A Rudder server can be used by different tenants, which are
+defined as a set of users and a set of node groups.
```
Comment (fanf) [inline release-data/changelogs/8.1/index.adoc:112]: Users or api tokens

**Pull request merged**